mediator/src/main/scala/org/hyperledger/identus/mediator/db/DataModels.scala

```diff
@@ -13,26 +13,33 @@ type HASH = String
 // messages
 type XRequestID = String // x-request-id
 
+enum MessageType {
+  case Mediator, User
+}
+
 case class MessageItem(
     _id: HASH,
     msg: SignedMessage | EncryptedMessage,
     headers: ProtectedHeader | Seq[SignProtectedHeader],
-    ts: String,
+    ts: Instant,
```

Comment:
Don't we need a migration script for this?
Let's just indicate one command that the user can run to migrate the DB to the next version